Info-Orbs/lib/config/config.h.template

```diff
@@ -82,12 +82,12 @@
 
 #define SHADOWING 1
 
-#define TIME_ZONE_OFFSET 2
+#define TIME_ZONE_OFFSET -4
 #define FORMAT_24_HOUR true
 
-#define WEATHER_LOCAION "Aberdeen,ID";
+#define WEATHER_LOCAION "47542";
 #define WEATHER_UNITS "us";
-#define WEATHER_API_KEY "XW2RDGD6XK432AF25BNK2A3C7"
+#define WEATHER_API_KEY "L8P9DU7SYN3DZ7QEBWY7YB8FR"
```

Comment:
The exiting api key is Brett's that he wanted to stay hard coded.

Reply:
I agree... I added mine before I saw that post.